core/db/migrate/20180404185904_add_default_state_to_shipment.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class AddDefaultStateToShipment < ActiveRecord::Migration[5.1]
+  def change
+    change_column_default(:spree_shipments, :state, Spree::Shipment::PENDING)
```

Comment:
We usually don't want to rely on code inside the migrations. Although very unlikely that the value for `Spree::Shipment::PENDING` will change ever, it still is more reliable to use the value here instead.